using-timescaledb/distributed-hypertables.md

```diff
@@ -48,32 +54,243 @@ CREATE TABLE conditions (
+fetching the raw data from each data node and then performing any
+necessary computations locally. Therefore, queries that can take
+advantage of push down and involve many data nodes are more likely to
+see better performance on a distributed hypertable compared to a
+regular hypertable. In particular, an aggregate computation that
+down-samples data is a good candidate for push down since it both
+distributes the computational workload across data nodes and reduces
+the amount of data that needs to be transferred in the result set.
+
+The planner can consider two types of push downs for aggregates:
+
+1. **FULL**: a full push down offloads all the computation to data
+   nodes and the access node only appends the individual results. To
+   enable full push down, the `GROUP BY` clause must include *all* the
+   partition keys (dimension columns) or *only* the first space
+   partition key. An example is computing the `max` temperature for
+   each location (`office_1`, `office_2`, etc.):
+
+   ```sql
+   SELECT location, max(temperature) FROM conditions
+   GROUP BY location;
+   ```
+
+   Given that we use `location` as a space partition, each data node
+   can compute the max on its own distinct subset of the data.
 
-This creates a multi-dimensional distributed hypertable
-partitioned along `time` and `location`.
-You can now insert data into the distributed hypertable and
-it will automatically be partitioned on the available data nodes
-by using the provided space partition. You
-can find more information for how to work with data in hypertables in the
-section [Creating Hypertables][creating-hypertables].
+1. **PARTIAL**: if a full push down is not possible, the planner will
+   do a partial push down instead. In this case, it offloads most of
+   the computation to data nodes, yielding a partial result that is
+   sent back to the access node and finalized by combining all the
+   partials from each data node. An example is computing the `max`
+   temperature; each data node computes a local max and then the
+   access node finalizes the result by computing the `max` over all
+   the data nodes' `max` results:
 
-### Changing the Number of Data Nodes for a Distributed Hypertable [](changing-data-nodes)
+   ```sql
+   SELECT max(temperature) FROM conditions;
+   ```
 
-By default, distributed hypertables will use all available data nodes when
-created. But, if you add new data nodes to the distributed database, they are
-not automatically added to existing distributed hypertables, and you need to
-explicitly attach them using [`attach_data_node`][attach_data_node].
+>:TIP: For push down to work, the settings variable
+>[`enable_partitionwise_aggregate`][partitionwise_agg_guc] must be set
+>to `TRUE` on the access node (the default is `FALSE`).
+
+Other types of push downs can include sorting operations, groupings,
+and joins. Push down for joins is currently not supported, however. To
```

Comment:
Yah, here too, not sure if it's a verb or noun...it seems to be used interchangeably